doc/source/ray-core/actors.rst

```diff
@@ -89,6 +89,27 @@ that specific worker and can access and mutate the state of that worker.
           // as the argument.
           auto counter = ray::Actor(CreateCounter).Remote();
 
+
+
+From Ray 2.0 onwards, you can use the :ref:`State API <state-api-overview-ref>`  to query for information about actors, or access the Actors View in the :ref:`Dashboard <observability-getting-started>` 
```

Comment:
Actually it seems more natural to make this part simple. Let's just do this?

"Use `ray list tasks <that has a link to the CLI>` to see a list of tasks that run". 

```
 # This API is only available when you install Ray with `pip install "ray[default]"`.
ray list tasks
```

```
  ======== List: 2023-05-25 10:10:50.095099 ========
  Stats:
  ------------------------------
  Total: 1
  
  Table:
  ------------------------------
      ACTOR_ID                          CLASS_NAME    STATE      JOB_ID  NAME    NODE_ID                                                     PID  RAY_NAMESPACE
   0  9e783840250840f87328c9f201000000  Counter       ALIVE    01000000          13a475571662b784b4522847692893a823c78f1d3fd8fd32a2624923  38906  ef9de910-64fb-4575-8eb5-50573faa3ddf
```